src/DependencyInjection/Compiler/TemplatingValidatorPass.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\RoutingBundle\DependencyInjection\Compiler;
+
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * To avoid a BC-Break: If templating component exists, we will use the validator using general templating
+ * from FrameworkBundle.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class TemplatingValidatorPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (class_exists(EngineInterface::class) && $container->has('templating')) {
```

Comment:
`templating` could exists in FrameworkBundle (before Symfony 3.2) and in a single namespace (3.2 and later). The one in FrameworkBundle is the one we need to use, as it is the one we used before. The one in the new component looks same but got an other namespace and would break the contructor then. So i will cover with this check the former `templating` to avoid a BC-Break. From Symfony 3.2 the user should use TwigBundle. But i have no clue how to test that one except from checking it out in sandbox with different symfony versions, any other idea?